multimedia/mythtv.28/Portfile

```diff
@@ -639,7 +639,7 @@ if {$subport eq "mythtv-plugins${majorversion}"} {
 
 if {$subport eq "mythtv${majorversion}"} {
     version             0${majorversion}${minorversion}-Fixes-${lastcommit}
-    revision            1
+    revision            2
```

Comment:
The mythtv.28 port doesn't need to be bumped but the mythtv-core.28 and mythtv-plugins.28 subports do.